modules/fedimint-ln-server/src/lib.rs

```diff
@@ -277,27 +292,35 @@ impl ServerModule for Lightning {
     }
 
     async fn await_consensus_proposal(&self, dbtx: &mut ModuleDatabaseTransaction<'_>) {
-        if !self.consensus_proposal(dbtx).await.forces_new_epoch() {
-            std::future::pending().await
+        while !self.consensus_proposal(dbtx).await.forces_new_epoch() {
+            sleep(Duration::from_millis(1000)).await;
```

Comment:
We do the same in the wallet module, it's because new items can emerge spontaneously from bitcoin rpc's height changing.  Eventually we will eliminate this kind of logic with the new consensus algo.